src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs

```diff
@@ -96,34 +96,61 @@ private void LoadingStatusIndicator_PropertyChanged(object sender, PropertyChang
             });
         }
 
-        // Indicates wether check boxes are enabled on packages
-        private bool _checkBoxesEnabled;
+        public bool CheckBoxesEnabled { get; set; }
```

Comment:
Any time I see these, I immediately go to thinking we should use databinding here. 

Is that something you considered but chose to go a different direction?

Reply:
I took a stab. We do use databinding, but it's not triggered for every code path. I'm not solving the world in this PR ;)

Reply:
Makes sense. 
Just curious.